### ISSUE TITLE
adding verbose during training

### ISSUE THREAD

**Issue by p9anand** [microsoft/interpret#7]
Can we see performance of model on validation data during training, like we have in xgboost.

**Comment by interpret-ml** [comment]
Seems like it could be a useful feature. We'll look into showing some real time metrics with a verbose flag, maybe as an experimental feature to see how it works. Thanks for suggesting this!

**Comment by ManuelZ** [comment]
Yes please, I just installed it and I have no idea if it's stuck or if it is really doing some useful work. That, alongside the huge memory consumption makes me wonder if I should have already killed it 10 mins ago.

**Comment by paulbkoch** [comment]
We now have callback functions that you can use to report this if you want it.
